medical/__openerp__.py

```diff
@@ -37,7 +42,7 @@
 ##############################################################################
 {
     'name': 'Odoo Medical',
-    'version': '8.0.1.1.0',
+    'version': '9.0.2.0.0',
```

Comment:
It's not a downgrading, because there's no internal versioning. Odoo only considers full version (for migration scripts, display...). The short version number is only a technical trick for not writing all, but we at OCA also remove that possibility by convention. The purpose of reset that part is to not have a module with version 12.0.24.1.0 in short.